src/Model/Translatable/TranslatableMethodsTrait.php

```diff
@@ -163,7 +164,6 @@ protected function doTranslate(?string $locale = null, bool $fallbackToDefault =
 
         $class = static::getTranslationEntityClass();
 
-        /** @var TranslationInterface $translation */
```

Comment:
Prevent this issue:
![Capture d’écran de 2020-02-04 07-20-24](https://user-images.githubusercontent.com/2103975/73719164-d49ef680-471e-11ea-9229-b788a33248cf.png)


Not a big deal, this modification is inside `doTranslate()` method which is protected

Reply:
Must be reverted